spec/api/v1/restrooms_spec.rb

```diff
@@ -12,9 +12,8 @@
     json.each do |rest_json|
       restroom = Restroom.find(rest_json['id'])
       expect(restroom.valid?).to be true
-      if previous_record.present?
-        expect(restroom.created_at).to be >= previous_record.created_at
-      end
+      # TODO: this assertion doesn't seem to ever run
+      expect(restroom.created_at).to be >= previous_record.created_at if previous_record.present?
```

Comment:
This `expect` didn't seem to run before these changes.

Also if you're open to it, I wouldn't mind adding something like `expect(json).not_to be_empty` above line 12 in order to prevent false positives if this loop never runs.

Reply:
I think adding `expect(json).not_to be_empty` sounds good. Thanks for noticing this!

Is the `# TODO` comment still accurate? Or if this `expect()` is running now, then I suppose the comment can/should be deleted.